controllers/operator/configmap.go

```diff
@@ -187,6 +187,7 @@ func (r *AuthenticationReconciler) handleConfigMap(instance *operatorv1alpha1.Au
 						return err
 					}
 					if configMapList[index] == "platform-auth-idp" {
+						newConfigMap.Data["MASTER_PATH"] = "/idauth"
```

Comment:
If this is always being set, this should just be set directly in the `authIdpConfigMap` function.